packages/loot-core/src/server/db/index.ts

```diff
@@ -147,19 +160,28 @@ export function asyncTransaction(fn: () => Promise<void>) {
 // async. We return a promise here until we've audited all the code to
 // make sure nothing calls `.then` on this.
 export async function all(sql, params?: (string | number)[]) {
-  return runQuery(sql, params, true);
+  // TODO: In the next phase, we will make this function generic
+  // and pass the type of the return type to `runQuery`.
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  return runQuery(sql, params, true) as any[];
 }
 
 export async function first(sql, params?: (string | number)[]) {
   const arr = await runQuery(sql, params, true);
-  return arr.length === 0 ? null : arr[0];
+  // TODO: In the next phase, we will make this function generic
+  // and pass the type of the return type to `runQuery`.
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  return arr.length === 0 ? null : (arr[0] as any);
```

Comment:
This will be made generic on the the next PRs so that we can control the PR size